src/modules/deleteTask.js

```diff
@@ -0,0 +1,9 @@
+import getTasks from './getTasks';
+
+const deleteTask = (id) => {
+  let tasks = getTasks();
+  tasks = tasks.filter((task) => Number(task.id) !== Number(id));
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+export default deleteTask;
```

Comment:
- [ ] According to the requirements, Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. as you can see in the below print screen that is not the case right now.
![afbeelding](https://github.com/10-menachi/todo-list/assets/39240820/da232871-4f98-4816-9a71-9fbb1a457262)